infrastructure/modules/xc3/output.tf

```diff
@@ -30,3 +30,8 @@ output "private_ip" {
   description = "Private IP address of ec2 instance to push prometheus metrics"
   value       = aws_instance.this.private_ip
 }
+
+output "load_balancer_dns" {
+ description = "DNS of the load balancer"
+ value       = var.domain_name != "" ? var.domain_name : aws_lb.this.dns_name
```

Comment:
remove this domain name condition from here

Reply:
Added in c250f7e1a2865da183d659363fa7e938f01ac440

Reply:
it is still there and don't resolve conversation by yourself

Reply:
same ^^

Reply:
![image](https://github.com/XgridInc/xc3/assets/122358742/32470594-7ce5-4fbb-b16a-5f022000ed58)
Its already done. Don't know why you are seeing this.